docs/Configuration.md

```diff
@@ -313,34 +313,6 @@ Default: `undefined`
 This option allows the use of a custom global teardown module which exports an
 async function that is triggered once after all test suites.
 
-### `mapCoverage` [boolean]
-
-##### available in Jest **20.0.0+**
-
-Default: `false`
-
-If you have [transformers](#transform-object-string-string) configured that emit
-source maps, Jest will use them to try and map code coverage against the
-original source code when writing [reports](#coveragereporters-array-string) and
-checking [thresholds](#coveragethreshold-object). This is done on a best-effort
-basis as some compile-to-JavaScript languages may provide more accurate source
-maps than others. This can also be resource-intensive. If Jest is taking a long
```

Comment:
The theory is that since `babel-jest` now returns `{map, source}`, it's not as intensive since we do not have to extract the inline sourcemap, meaning there is no reason to opt out of this behavior